app/src/main/java/org/sopt/dosopttemplate/util/Extensions.kt

```diff
@@ -22,7 +21,7 @@ fun showShortSnackBar(view: View, text: String) {
     ).show()
 }
 
-fun hideKeyboard(view: View) {
-    val imm: InputMethodManager? = null
-    imm?.hideSoftInputFromWindow(view.windowToken, 0)
-}
+// fun hideKeyboard(view: View) {
+//    val imm: InputMethodManager? = null
+//    imm?.hideSoftInputFromWindow(view.windowToken, 0)
+// }
```

Comment:
```suggestion
fun Context.hideKeyboard(view: View) {
    val inputMethodManager = getSystemService(Activity.INPUT_METHOD_SERVICE) as? InputMethodManager
    inputMethodManager?.hideSoftInputFromWindow(view.windowToken, 0)
}
```

최상위 함수로 빼려는 시도 너무 좋습니다👍 저는 이런식으로 확장함수로 빼놓았어요!

Reply:
시도했다가 적용이 안되어서 주석처리 해놨는데 참고해서 다시 시도해볼게요 ㅠ~ㅠ